### PR TITLE
Dalton gui polishing

### PR DESCRIPTION
**Pull request by DaltonWemer**
The UI is complete aside from one minor bug that does not impact the use of the software. I am open to suggestions to update any of the animations if you think it could look better!

### CODE REVIEW

**Comment by CalebAlex-Ander** [comment]
What is the bug that occurs?

File: electron/home.js [hunk 1]
```diff
@@ -299,38 +296,43 @@ async function startRecording() {
         mediaRecorder.mimeType = 'audio/wav'; // check this line for audio/wav
         mediaRecorder.audioChannels = 1;
         mediaRecorder.sampleRate = 44100;
+        
+        // Set the max length of recording
+        var maxLength = 5
```
Comment (CalebAlex-Ander) [inline electron/home.js]: consider making this a constant

File: electron/home.js [hunk 2]
```diff
@@ -62,39 +72,91 @@ window.onload = function () {
...
 }
 
 async function watchForError() {
     fs.watch(error_log_path, (eventType, filename) => {
         if (eventType == 'change') {
-            document.getElementById("result").innerHTML = "error, check logs";
-            document.getElementById("result-container").style.visibility = "visible";
-            document.getElementById("result-container").style.backgroundColor = "#EE4B2B";
+            document.getElementById("loadingAnimation").style.display = "none"
+            // document.getElementById("result").innerHTML = "error, check logs";
```
Comment (CalebAlex-Ander) [inline electron/home.js:130]: Consider removing all commented out code when pushing to the main branch

File: records/archive/result_logs/Nov-08-2021_17-14-23.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Classification Date and Time: Nov-08-2021_17-14-23
```
Comment (CalebAlex-Ander) [inline records/archive/result_logs/Nov-08-2021_17-14-23.txt:1]: These files aren't necessary to the pull request/main branch, consider adding them and/or anything in this directory to .gitignore

**Review by CalebAlex-Ander — DISMISSED** [review]
All in all, a good pull, if you have time I'd recommend slightly modifying your git ignore file as well as a few things here and there. They aren't necessary, but I thought I'd be knit picky for the sake of a good pull request review.

**Review by sethswango — APPROVED** [review]

**Review by brendanbeck — APPROVED** [review]

**Pull request merged**